dotCMS/src/main/webapp/WEB-INF/messages/Language.properties

```diff
@@ -4620,12 +4620,14 @@ system.locale.language.inputs.language.options.zh=Chinese
 system.locale.language.inputs.language.options.zu=Zulu
 
 ## Content Types
-contenttypes.content.file=File
+contenttypes.content.fileasset=File
 contenttypes.content.content=Content
 contenttypes.content.form=Form
 contenttypes.content.persona=Persona
 contenttypes.content.widget=Widget
-contenttypes.content.page=Page
+contenttypes.content.htmlpage=Page
```

Comment:
hey are you sure that this message are being not referrer on any java code.
I guess they not, but just to double check!

Reply:
Those are new labels for the content types Angular module.